pages/index.js

```diff
@@ -37,10 +37,15 @@ function HomePage() {
     <>
       <Container wide>
         <Head />
-        <NavBar className="z-50" />
-        <Hero className="mb-24" />
       </Container>
```

Comment:
```
<Head>
```

component only adds metadata in the `<head>` tag so I think that you don't need wrap it in `<Container wide>` component. You can write standalone `<Head>` :)